updated conda section (#1204)
pypa/packaging.python.org: source/key_projects.rst

:doc:`Docs <conda:index>`

conda is a package, dependency, and environment management system for any language—Python, R,

> Ruby, Lua, Scala, Java, JavaScript, C/ C++, Fortran, and more. It is written in Python and

Author: *Any* language is not too general -- this is an important point -- conda can mange ANY binary package -- just like, e.g. rpm or dpkg.  This is a very important point that many in the Python community miss.

In practice, only some languages are currently being used for conda packages (e.g. in conda-forge) but that's community need, not a limitation of the conda system.

But shorten the enumeration -- sure.

> Ruby, Lua, Scala, Java, JavaScript, C/ C++, Fortran, and more. It is written in Python and

Author: Done in my repo.

widely used in the Python scientific computing community, due to its support for non-Python
compiled libraries and extensions. It is used as the basis of the `Anaconda
<https://docs.anaconda.com/anaconda/>`__ Python distribution from `Anaconda, Inc
<https://www.anaconda.com/products/individual>`__ specifically aimed at the scientific
community, but can also be used on its own, or with the
`miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ distribution. It is available
for Windows, Mac and Linux systems.

Conda is a completely separate tool from :ref:`pip`, virtualenv and wheel, but provides
many of their combined features, such as package management, virtual environment
management and deployment of binary extensions and other binary code.

> Conda does not install packages from PyPI -- it can only manage packages built specifically
> for conda, which can be made available on a "conda channel", such as those hosted by
> `anaconda.org <https://anaconda.org/>`_ or a local (e.g. intranet) package server. In additon
> to the "defults" channel managed by `Anaconda, Inc <https://www.anaconda.com/>`_, there are

Reviewer: ```suggestion
Conda does not install packages from PyPI -- it can only manage packages built specifically
for conda, which can be made available on a "conda channel", such as those hosted on
`anaconda.org <https://anaconda.org/>`_ or a local (e.g. intranet) package server. In additon
to the "default" channels managed by `Anaconda, Inc <https://www.anaconda.com/>`_, there are
```
